pkg/users/local.js

```diff
@@ -1193,7 +1193,16 @@ function AccountExpiration() {
     $('#account-expiration-input').datepicker({
         autoclose: true,
         todayHighlight: true,
-        format: 'yyyy-mm-dd'
+        language: (function() {
+            try {
+                if (cockit.language && cockpit.language !== 'en')
+                    require('bootstrap-datepicker/dist/locales/bootstrap-datepicker.' +
+                            cockpit.language.replace('_', '-') + '.min');
+            } catch (ex) {
+                console.warn('Setting datepicker locale failed', ex);
+            }
+            return cockpit.language;
```

Comment:
Please test this with pt_BR, it could be that it requires the POSIX→ JS format conversion as well.

Reply:
Tested this, and it only accepts JS format here. I'll change it to `return cockpit.language.replace('_', '-');`